source/includes/_cluster.md

```diff
@@ -353,6 +356,60 @@ Fetches the details of a node in the cluster.
 
 A `ClusterNode` resource.
 
+## Add Cluster Node
+
+> Request Example
+
+```python
+client.cluster.nodes.add(host_ip='172.20.0.3', node_name='second-manager')
+```
+
+```shell
+$ curl -u user:password -d '{"host_ip": "172.20.0.3", "node_name": "second-manager"}' "http://<manager-ip>/api/v3/cluster/nodes"
+```
+
+```javascript
+var headers = {
+   'content-type': 'application/json',
+   'authorization': 'Basic ' + new Buffer(username + ':' + password).toString('base64')
+}
+
+var settings = {
+  "url": "http://<manager-ip>/api/v3/cluster/nodes",
+  "method": "GET",
+  "headers": headers,
+  "contentType": "application/json"
+  "data": JSON.stringify({
+      "host_ip": "172.20.0.3",
+      "node_name": "second-manager"
+  })
+}
+
+$.ajax(settings).done(function (response) {
+  console.log(response);
+});
+```
+
+> Response Example
+
+```json
+{
+    "initialized": false,
+    "online": false,
+    "master": false,
+    "host_ip": "172.20.0.3",
+    "name": "second-manager",
+    "credentials": "<REDACTED>"
+}
+```
+
+`PUT "{manager-ip}/api/v3/cluster/nodes/{node-name}"`
+
+Adds a node to the cluster. This prepares the cluster for contacting the new node,
+runs validations and generates credentials for the new node to use. The received
+credentials should be passed in the "Join cluster" ("Put Cluster State") API
```

Comment:
1. ~for a new node to use~ **for use by a new node**
2. the credentials **are** passed